client/components/repository/WorkflowBoard/Columns/index.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="tasks">
+    <div v-if="workflow" class="column-layout mt-4 px-4">
```

Comment:
Isn't the workflow page shown only if workflow exists?

Reply:
It breaks if the user lands directly on the page because the repository is not yet loaded. Is there a nicer way to ensure that doesn't happen?

Reply:
Yep, there is a showLoader prop being passed from repository root container (`repository/index`) which is set to true until the repo is initialized.